### PR TITLE
Repurpose CidInfo

### PR DESCRIPTION
**Pull request by asutula**
`CidInfo` has always represented the current storage state of a cid. A better name for it is `StorageInfo`. An that frees up the name `CidInfo` to be used in the API as an object that provides general information about the state of a cid in the system.

### CODE REVIEW

File: ffs/scheduler/internal/cistore/cistore.go [hunk 1]
```diff
@@ -45,8 +45,8 @@ func (s *Store) Get(c cid.Cid) (ffs.CidInfo, error) {
 	return ci, nil
 }
 
-// Put saves a new storing state for a Cid.
-func (s *Store) Put(ci ffs.CidInfo) error {
+// Put saves a new storage state for a Cid.
+func (s *Store) Put(ci ffs.StorageInfo) error {
```
Comment (jsign) [inline ffs/scheduler/internal/cistore/cistore.go:49]: So we got lucky that the JSON serialization is naming agnostic.

**Review by jsign — APPROVED** [review]
LGTM. Getting a bit nervous about such a big PR and if we're missing some breaking change at the datastore layer; shouldn't be the case.

File: ffs/scheduler/internal/cistore/cistore.go (hunk 1 above)
Comment (asutula) [reply ffs/scheduler/internal/cistore/cistore.go:49]: Exactly ;)

**Comment by asutula** [comment]
I've really gone out of my way to make sure there are no breaking datastore changes in all this work. And all my remaining big changes are going to be strictly API related.... so I _thiiiink_ we're ok.

**Pull request merged**